lib/spec/router.sw-spec.js

```diff
@@ -206,4 +206,142 @@ describe('Router instances', function () {
     });
   });
 
+
+  describe('Matching algorithm', function() {
+    describe('match()', function() {
+      var mw = function() {};
+
+      it('should match universal registrations (*)', function() {
+        router.get('*', mw);
+
+        expect(router.match('get', '/')).to.deep.equal([mw]);
+        expect(router.match('get', '/foo/bar')).to.deep.equal([mw]);
+      });
+
+      it('should match all universal registrations (*) in order', function() {
```

Comment:
Reword to `should match all universal registrations (*) preserving registration order`